### DISCUSSION TITLE
Fonts using any of the sets

### DISCUSSION THREAD

**Discussion by StudioTriple** [koeberlin/Latin-Character-Sets#17] [Ideas]
I thought it would be neat to have a place here where people can post fonts they know embrace one of the character sets.

**Comment by StudioTriple** [comment]
And I start with [Unbounded](https://www.nan.xyz/custom/polkadot/) that supports the Latin L set.

**Comment by juliusross1** [comment]
Thanks for the work you put into these sets.  To let you know [Pennstander](https://github.com/juliusross1/Pennstander) now has coverage for Latin S and Latin M.

https://github.com/juliusross1/Pennstander/releases/tag/v0.4.1

Am sure there are lots of issues since it is hard to create characters without knowing context, but this is meant for fun/learning rather than anything else.  

Thanks again!

**Reply by koeberlin** [reply]
Great. Thank you, Julius! I’ll add it to the list. ✌️
